### PR TITLE
added baseline indices to baselines function.

### PR DESCRIPTION
**Pull request by mmphego**
added baseline indices to baselines function.

### CODE REVIEW

File: mkat_fpga_tests/test_cbf.py [hunk 1]
```diff
@@ -234,18 +234,36 @@ def calc_zero_and_nonzero_baselines(nonzero_inputs):
                         zeros.add((inp_i, inp_j))
             return zeros, nonzeros
 
-        #zero_baseline, nonzero_baseline = calc_zero_and_nonzero_baselines(nonzero_inputs)
+        zero_baseline, nonzero_baseline = calc_zero_and_nonzero_baselines(nonzero_inputs)
```
Comment (brickZA) [inline mkat_fpga_tests/test_cbf.py]: Perhaps zero_baseline -> zero_baselines, and nonzero_baselines, indicate that these variables contain multiple baselines.

File: mkat_fpga_tests/test_cbf.py [hunk 2]
```diff
@@ -234,18 +234,36 @@ def calc_zero_and_nonzero_baselines(nonzero_inputs):
...
         def print_baselines():
             print ('zeros: {}\n\nnonzeros: {}\n\nnonzero-baselines: {}\n\n '
                 'zero-baselines: {}\n\n'.format(
                     sorted(zero_inputs), sorted(nonzero_inputs),
                     sorted(nonzero_baseline), sorted(zero_baseline)))
-        #print_baselines()
+
         for inp in input_labels:
             old_eqs = initial_equalisations[inp]
```
Comment (brickZA) [inline mkat_fpga_tests/test_cbf.py]: Perhaps name this 'old_eq', i.e. singular since it is for one input only.

Comment (mmphego) [inline mkat_fpga_tests/test_cbf.py]: Fixed

File: mkat_fpga_tests/test_cbf.py (hunk 1 above)
Comment (mmphego) [inline mkat_fpga_tests/test_cbf.py]: Fixed

**Pull request closed without merging**